test/integration/load_node_test.cpp

```diff
@@ -78,6 +79,45 @@ TEST(LoadNodeTest, IterateMap) {
   EXPECT_EQ(3, i);
 }
 
+static void alias_access(Node&& doc) {
```

Comment:
If you do keep this function (see below):

- change its name to camel case
- put it in an anonymous namespace instead of being static
- make it take a const ref instead of an rvalue reference
- call it something that indicates it does some assertions, like VerifyAliasAccess